src/components/services/Events.vue

```diff
@@ -1,39 +1,113 @@
+<script>
```

Comment:
The script section should be below the template section. You might see some Vue 3 tutorials that have it above like this. But we should be consistent within the project. And all rest of the components have \<template> at the top, then <script> and <style> at the bottom.

The reason for being consistent is that it makes the code easier to understand for other developers that might work on a project.